api/test/pact/pact-tests/work-allocation-get-locations.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Pact } from '@pact-foundation/pact';
+import { assert, expect } from 'chai';
+import * as getPort from 'get-port';
+import * as path from 'path';
+
+import { EnhancedRequest } from '../../../lib/models';
+import { handleLocationGet } from '../../../workAllocation/locationService';
+
+describe('Work Allocation Location API', () => {
+
+  let mockServerPort: number
+  let provider: Pact
+
+  before(async () => {
+    mockServerPort = await getPort();
+    provider = new Pact({
+      consumer: 'xui_work_allocation_get_locations',
+      provider: 'WorkAllocation_api_location', // TODO: Need to clarify naming conventions here, as we're using different ones.
+      dir: path.resolve(__dirname, '../pacts'),
+      log: path.resolve(__dirname, '../logs', 'work-allocation.log'),
+      logLevel: 'info',
+      port: mockServerPort,
+      spec: 2
+    });
+    return provider.setup();
+  });
+
+  // Write Pact when all tests done
+  after(() => provider.finalize());
+
+  /**
+   * Note that the location body is yet to be defined, by us or WA Allocation team.
+   */
+  describe('should return locations.', () => {
+
+    const LOCATION_BODY = ['location', 'location'];
```

Comment:
Isn't the location body the one defined in `api/workAllocation/interfaces/task.ts`, line 30?

Reply:
You are right, I missed that, will put in the changes.

Reply:
You are right, I missed that, will put in the changes.

Reply:
@connorpgpmcelroy has kindly offered to put in the changes, thank you!

Reply:
Should be all good now